rosidl_typesupport_c/QUALITY_DECLARATION.md

```diff
@@ -2,9 +2,9 @@ This document is a declaration of software quality for the `rosidl_typesupport_c
 
```

Comment:
How about making the `linux-arm64_release` -> `linux_release` to this file too?

Reply:
Done